foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -51,7 +63,8 @@ def handle_batch(self, batch: Batch, key_values: BatchKeyValue = None):
         batch.survey_job = self.survey_job
         batch.source_type = self.source_type()
         batch.status = BatchStatuses.NEW.value
-        batch.internal_location = None
+        batch.internal_location = (self.source_type() + "/"
+                                   + batch.accession_code + "/")
```

Comment:
So I'm definitely all for using `os.path.join()` and have in fact already made the change locally, however how big of an issue do you think portability is given that this project is being build around being run in a docker container? For example [here](https://github.com/data-refinery/data_refinery/pull/8/files#diff-5185347fdc47e66538c13d8187765e49R6) I list the root path as a Linux path which is pointing to a specific place within the docker container. I could turn that into an environment variable in order to make it easier to change based on the environment, however I plan on keeping that location within the Docker container consistent across environments.